[WIP]README記述

matsufuji06 opened the pull request:
#What
本アプリケーションDB設計を行う。

#Why
本アプリケーションにおいて必要なデータベースやその中のカラムや制約を整理していくため。

matsufuji06 committed:
first commit

matsufuji06 committed:
README_08_19

shingoteshima commented on README.md:
```diff
@@ -22,3 +22,110 @@ Things you may want to cover:
+|Column|Type|Options|
+|------|----|-------|
+|user_id|references|null: false, foreign_key: true|
+|category_id|references|null: false, foreign_key: true|
```
category_idを持つのであればbelongs_to :categoryとなります。

matsufuji06 committed:
modify_README_on_08_20_1

We deleted "|category_id|references|null: false, foreign_key: true|" written in README.md, line 93.

matsufuji06 commented:
#再送いたしました。
|category_id|references|null: false, foreign_key: true|
を削除し、リレーション はそのままです。
ご確認をよろしくお願いいたします。

shingoteshima commented on README.md:
```diff
@@ -22,3 +22,109 @@ Things you may want to cover:
+
+### Association
+- has_many :items,
+- has_many :items_categores, through: :items
```
usersテーブルに44行目は必要でしょうか？

matsufuji06 committed:
modify_README_on_08_30_2

We deleted "- has_many :items_categores, through: :items" written in README.md, line44

matsufuji06 commented:
修正分をお送りいたします。
なお、Trelloの注意事項に「プルリクエストの際は、ER図も添付すること」という記載があることから、
下記にてER図のキャプチャ画像のリンク先を添付しているのですが、これをもって"添付"と見なしていただけるのでしょうか？
お手数おかけしますが、何卒よろしくお願いいたします。
↓↓
[https://gyazo.com/5c3e6417a979458669fd96e44ba88469](url)

shingoteshima commented:
キャプチャありがとうございます！
LGTMです！！
今後実装していく中で変更点など出てくると思いますので、その都度修正を加えていきましょう！

matsufuji06 committed:
modify_README_on_08_30_3

We modified "," at line 43.

matsufuji06 commented:
@shingoteshima 
了解いたしました！大変お世話おかけいたしました！
引き続きよろしくお願いいたします！
